workflows/pipe-common/shell/install_gcsfuse

```diff
@@ -57,6 +57,57 @@ EOF
     fi
 }
 
+function check_installed {
+      local _COMMAND_TO_CHECK=$1
+      command -v "$_COMMAND_TO_CHECK" >/dev/null 2>&1
+      return $?
+}
+
+function local_package_install {
```

Comment:
It may make sense to move the `local_package_install` to a separate script, that can be further reused? We'll definitely perform such operations for other components

Reply:
good point